lib/pharos/command_options/tf_json.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Pharos
+  module CommandOptions
+    module TfJson
+      def self.included(base)
+        base.prepend(InstanceMethods)
+
+        base.option '--tf-json', 'PATH', 'path to terraform output json' do |config_path|
+          File.realpath(config_path)
+        rescue Errno::ENOENT
+          signal_usage_error 'File does not exist: %<path>s' % { path: config_path }
+        end
+      end
+
+      module InstanceMethods
+        private
+
+        # @param config_hash [Hash]
+        def load_external_config(config_hash)
+          load_terraform(tf_json, config_hash) if tf_json
+        end
+
+        # @param file [String]
+        # @param config [Hash]
+        # @return [Hash]
+        def load_terraform(file, config)
+          puts(pastel.green("==> Importing configuration from Terraform ...")) if $stdout.tty?
+
+          tf_parser = Pharos::Terraform::JsonParser.new(File.read(file))
+          config['hosts'] ||= []
+          config['api'] ||= {}
+          config['addons'] ||= {}
+          config['hosts'] += tf_parser.hosts
```

Comment:
```suggestion
          config['hosts'].concat(tf_parser.hosts)
```
would avoid intermediate array